test/CMakeLists.txt

```diff
@@ -13,6 +13,10 @@ set(DLAF_MPI_PRESET "plain-mpi" CACHE STRING "Select a preset to use")
 set_property(CACHE DLAF_MPI_PRESET PROPERTY STRINGS ${DLAF_PRESET_OPTIONS})
 option(DLAF_CI_RUNNER_USES_MPIRUN "Remove mpiexec command for tests executed by ctest. This option is to be used if the CI runner executes the tests with <mpiexec + options> ctest -L RANK_<spawned MPI ranks>" OFF)
 
+# On some machines, tests using multiple ranks + oversubscribing run
+# significantly faster when threads are not pinned.
+option(DLAF_TEST_DISABLE_THREAD_BINDING "If ON disables HPX thread binding." OFF)
```

Comment:
`DLAF_TEST_ENABLE_THREAD_BINDING` and default to `ON`? Not a big deal, I just prefer not having to do double negation in my head :P

Reply:
Sounds good! I just changed it to `DLAF_TEST_THREAD_BINDING_ENABLED`.